### PR TITLE
rpki-client 7.6

### PR DESCRIPTION
**Pull request by geeohgeegeeoh**
Upgrade rpki-client to 7.6 from OpenBSD

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
I guess the test might need updating.
```
==> /opt/homebrew/Cellar/rpki-client/7.6/sbin/rpki-client -n -d . -R . 2>&1 | tail -1
Error: rpki-client: failed
An exception occurred within a child process:
  Minitest::Assertion: Expected /fts_read\ ta:/ to match "rpki-client: parse file ta/ripe/ripe-ncc-ta.cer: No such file or directory\n".
```

**Comment by geeohgeegeeoh** [comment]
Yea. I missed that. Sorry, revised the stringmatch to the new error msg from the client.

File: Formula/rpki-client.rb [hunk 1]
```diff
@@ -34,6 +34,6 @@ def post_install
   end
 
   test do
-    assert_match "fts_read ta:", shell_output("#{sbin}/rpki-client -n -d . -R . 2>&1 | tail -1")
+    assert_match "parse file ta:", shell_output("#{sbin}/rpki-client -n -d . -R . 2>&1 | tail -1")
```
Comment (carlocab) [inline Formula/rpki-client.rb:37]:
```suggestion
    assert_match "parse file ta:", shell_output("#{sbin}/rpki-client -n -d . -R . 2>&1").lines.last
```

**Comment by carlocab** [comment]
Please keep merge commits out of your pull request. @BrewTestBot isn't able to merge PRs with merge commits given the repo's linear history requirement, which is why there was an `automerge-skip` label. I've cleaned that up for you.

**Comment by geeohgeegeeoh** [comment]
Ah sorry. Apologies for not sticking to one process.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**